firmware/application/hw/debounce.cpp

```diff
@@ -149,3 +149,24 @@ bool Debounce::feed(const uint8_t bit) {
 
     return false;
 }
+
+uint8_t EncoderDebounce::state() {
+    return state_;
+}
+
+// Returns TRUE if encoder position phase bits changed (after debouncing)
+bool EncoderDebounce::feed(const uint8_t phase_bits) {
```

Comment:
Note to reviewers:  How much simpler this function is when compared to the Debounce function used for switches!  No need to support Repeat or Long Press for the encoder input bits, and both encoder bits are now debounced at once for better deglitching.